docs/rfc-1-staking-contract.adoc

```diff
@@ -369,27 +342,21 @@ each affected application.
 
 Returns the authorized stake amount of the staking provider for the application.
 
-==== `stakes(address stakingProvider) external view returns (uint96 tStake, uint96 keepInTStake, uint96 nuInTStake)`
```

Comment:
See my comment above about leaving this method but marked as deprecated

Reply:
✔️